bin/test.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+
+set -ea
+
+cmd_usage="Run test suite
+
+Usage: run-test <seed> <faucet-private-key> [options]
```

Comment:
test. Let's make `faucet-private-key` as an option. I have a feeling that we could get rid of this dependency later.